patches/corefx/0001-Skip-CoreFx.Private.TestUtilities-in-source-build.patch

```diff
@@ -14,11 +14,11 @@ Patch PR: https://github.com/dotnet/corefx/pull/33217
  2 files changed, 9 insertions(+), 1 deletion(-)
 
 diff --git a/Directory.Build.props b/Directory.Build.props
-index eb59fd5c25..3fdd594ffa 100644
+index 9d96e84da0..9c300dabaa 100644
 --- a/Directory.Build.props
 +++ b/Directory.Build.props
-@@ -320,6 +320,14 @@
-     <IncludeDllSafeSearchPathAttribute Condition="'$(TargetGroup)' == 'netstandard1.0'">false</IncludeDllSafeSearchPathAttribute>
+@@ -401,6 +401,14 @@
+     <VersionFileForPackages Condition="'$(VersionFileForPackages)' == ''">$(ArtifactsObjDir)version.txt</VersionFileForPackages>
```

Comment:
I don't understand this patch and why it changed from one thing to something completely different?

Reply:
This is just the patch's context changing when I updated it from one commit to another--the patch is the same, but diffs of diffs are really confusing. Looking at the patch itself is much clearer: https://github.com/dotnet/source-build/blob/6190f47699b9b0dfef549de3e72e5766ce4a3612/patches/corefx/0001-Skip-CoreFx.Private.TestUtilities-in-source-build.patch

```patch
@@ -401,6 +401,14 @@
     <VersionFileForPackages Condition="'$(VersionFileForPackages)' == ''">$(ArtifactsObjDir)version.txt</VersionFileForPackages>
   </PropertyGroup>
 
+  <!--
+    Remove test-only dependencies during source-build: disable project that brings in XUnit.Runtime
+    dependencies through 'ReferenceFromRuntime' property.
+  -->
+  <ItemGroup Condition="'$(DotNetBuildFromSource)' == 'true' and '$(BuildTests)' != 'true'">
+    <ProjectExclusions Include="$(SourceDir)CoreFx.Private.TestUtilities/**/*.csproj" />
+  </ItemGroup>
+
   <!-- Properties related to multi-file mode for ILC tests -->
   <PropertyGroup Condition="'$(TargetGroup)' == 'uapaot'">
     <!-- Only enable multi-file for Release-x64 and Debug-x86 for now -->
```

Reply:
Oh OK, Got it.  Thanks!